dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -900,8 +887,7 @@ public void reorderAction(final WorkflowReorderBean workflowReorderActionStepFor
         }
 
         try {
-            Logger.debug(this, () -> "Looking for the stepId: "
-                    + workflowReorderActionStepForm.getStepId());
+            Logger.debug(this, () -> "Looking for the stepId: " + workflowReorderActionStepForm.getStepId());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Looking for the stepId: " appears 5 times in this file; the first occurrence is on line 890](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)